src/panel_gwalker/_gwalker.py

```diff
@@ -84,63 +82,45 @@ class GraphicWalker(ReactComponent):
         Servable `GraphicWalker` object that creates a UI for visual exploration of the input DataFrame.
     """
 
-    object = param.DataFrame(doc="""The data to explore.""")
-    fields = param.List(doc="""Optional fields, i.e. columns, specification.""")
-    appearance = param.Selector(
-        default=None,
-        objects=[None, "media", "dark", "light"],
-        doc="""Dark mode preference: 'media', 'dark', 'light' or None.
-        If None the the appearance is derived from pn.config.theme""",
+    object: pd.DataFrame = param.DataFrame(
+        doc="""The data to explore.
+        Please note that if you update the `object`, then the existing charts will not be deleted."""
     )
-
-    config = param.Dict(
-        doc="""Optional extra Graphic Walker configuration. See the
+    fields: list = param.List(doc="""Optional fields, i.e. columns, specification.""")
+    appearance: Literal["media", "dark", "light"] = param.Selector(
+        default="light",
+        objects=["light", "dark", "media"],
+        doc="""Dark mode preference: 'light', 'dark', 'media'.
+        If not provided the appearance is derived from pn.config.theme.""",
+    )
+    # This one is added to better explain that currently only 'client' mode is supported
+    # but we envision supporting 'server' mode one day
+    computation: Literal["client"] = param.Selector(
+        objects=["client"],
+        doc="""The computation configuration. Currently only 'client' is supported.""",
+    )
+    config: dict = param.Dict(
+        doc="""Optional extra Graphic Walker configuration. For example `{"i18nLang": "ja-JP"}`. See the
     [Graphic Walker API](https://github.com/Kanaries/graphic-walker#api) for more details."""
     )
 
+    current_chart: dict = param.Dict(doc="""The current chart.""")
+    export_current_chart: bool = param.Event(doc="""Updates the current chart.""")
+
+    current_chart_list: list = param.List(doc="""The current chart list.""")
```

Comment:
Not sure the `current_` prefix adds much semantically to all these parameter names.

Reply:
Again. This is close to the original JS api. That is why I chose the name.

Please let me know if I should change? Thanks

Reply:
I chose it because it was similar to the JS naming convention.

On a second though I will change to `chart` and `export_chart`.